src/resources/js/views/scripts.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since  TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events = tribe.events || {};
+tribe.events.views = tribe.events.views || {};
+
+/**
+ * Configures Views Object in the Global Tribe variable
+ *
+ * @since  TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.events.views.scripts = {};
```

Comment:
Let's create one file for each script, e.g.: `tooltip`, `datepicker`, `month-day-picker`